Trustroots/trustroots
# Reactify header

#### Proposed Changes

* Rewrite header in React, allowing it to be translated

<img width="1005" alt="Screenshot 2021-04-03 at 19 08 01" src="https://user-images.githubusercontent.com/87168/113484339-78656d00-94b0-11eb-9305-2179af81527f.png">


#### Testing Instructions

* Ensure header collapses and functions in different screen widths
* Ensure dropdowns work
* Ensure active pages get marked as "active" visually; inside dropdowns, I didn't implement this as it was a bit elaborate. We can get this feature back with a different, React-based router easier.
* Ensure keyboard navigation works (walking through links and dropdowns with tab/arrows/enter)

Part of https://github.com/Trustroots/trustroots/issues/1334
